### DISCUSSION TITLE
如何让Mortal VS Mortal ？

### DISCUSSION THREAD

**Discussion by linforgithub** [Equim-chan/Mortal#66] [Q&A]
我想使用Mortal和Mortal对战，将对战的结果作为训练集，但是不知道这个应该需要怎么设置，才可以自动对战。我看了config.example.toml里有好多的配置文件，这些文件内容是需要写入啥呢？

**Comment by Equim-chan** [comment]
要先有 Mortal 模型才能对战。目前这部分的文档还没有整理。

**Reply by linforgithub** [reply]
那就是我需要先爬一些牌谱来训练，之后才可以是么。
最近是否有计划整理文档呢，没学过深度学习，看着代码有点吃力o(╥﹏╥)o

**Comment by DokiDoki1103** [comment]
作者没有公布模型吗？

**Comment by huanghaiji666666** [comment]
什么时候公布模型,是否有这个打算?我想批量跑谱然后用python处理数据到excel上面,同步牌谱屋,打一局之后就会有本局的ai一致率和得分
但是我想到可能会被别有用心的人拿去开挂,所以不公开也确实是正确的选择
